.gitignore

```diff
@@ -1,4 +1,14 @@
 dist/
 node_modules/
-.vscode/
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+
+# Created by https://www.gitignore.io/api/visualstudiocode
+
+### VisualStudioCode ###
+.vscode/*
+!.vscode/settings.json
```

Comment:
Which one of these should we keep?
```
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```

I'm assuming at least extensions.json so that we can have .editorconfig in there. ;)